packages/styles/combobox.css

```diff
@@ -111,8 +111,28 @@
   position: relative;
 }
 
+.Combobox .TextFieldWrapper {
+  flex-wrap: wrap;
+}
+
 .Combobox .TextFieldWrapper input[type='text'] {
   margin-right: 1.25rem;
+  flex-grow: 1;
+  width: unset;
+  min-width: var(--input-min-width);
+}
+
+.Combobox__selected-tag {
+  margin-block: var(--space-smallest);
```

Comment:
I'm not sure this is correct as the vertical spacing between the selected items doesn't appear to match the designs.

Actual:
![image](https://github.com/dequelabs/cauldron/assets/1062039/35ef2f14-11b5-4a73-9894-1da19d9c003b)

Proposed Design:
![image](https://github.com/dequelabs/cauldron/assets/1062039/782682c5-2d70-41ea-8813-da290ddcd1e1)

The down carat is also not positioned correctly in the top right.